modules/querying/pages/query-operations.adoc

```diff
@@ -970,9 +1011,64 @@ This statement sets a persistent system parameter. Each version of the TigerGrap
 showQuery := SHOW QUERY queryName
 ----
 
-To show the GSQL text of a query, run `SHOW QUERY query_name` . The `query_name` argument can use `*` or `?` wildcards from Linux globbing, or it can be a regular expression when preceded by  `-r`. See  xref:ddl-and-loading:defining-a-graph-schema.adoc#_show__view_parts_of_the_catalog[SHOW: View Parts of the Catalog]
+To show the GSQL text of a query, run `SHOW QUERY query_name` . The `query_name` argument can use `*` or `?` wildcards from Linux globbing, or it can be a regular expression when preceded by  `-r`.
+See  xref:ddl-and-loading:defining-a-graph-schema.adoc#_show__view_parts_of_the_catalog[SHOW: View Parts of the Catalog]
+
+Additionally, the `ls` GSQL command lists all created queries and identifies which queries have been installed as well as their draft or valid status.
```

Comment:
`ls` command not only shows `Queries` section, but contains `Vertex Types`, `Edge Types`, etc.
and identifiers contains `draft`, `installed v2`, `deprecated`, `blocked`, `disabled`, etc.